Taarafo.Portal.Web/Models/Groups/Exceptions/GroupServiceException.cs

```diff
@@ -0,0 +1,18 @@
+// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using Xeptions;
+
+namespace Taarafo.Portal.Web.Models.Groups.Exceptions
+{
+    public class GroupServiceException : Xeption
+    {
+        public GroupServiceException(Exception innerException)
```

Comment:
Xeption innerException

Reply:
I changed the Exception to Xeption in the PR titled "CODE RUB: Changed Exception to Xeption."